src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -108,16 +111,21 @@ public function setProxyDumper(ProxyDumper $proxyDumper)
     public function dump(array $options = array())
     {
         $this->targetDirRegex = null;
+        $this->inlinedRequires = array();
         $options = array_merge(array(
             'class' => 'ProjectServiceContainer',
             'base_class' => 'Container',
             'namespace' => '',
             'as_files' => false,
             'debug' => true,
+            'hot_path_tag' => null,
+            'inline_class_loader_parameter' => 'container.dumper.inline_class_loader',
```

Comment:
Why is it configurable? Usually we don't want to add new options. IMHO, this is useless here.

Reply:
I don't like hardcoding a parameter name into the dumper